### PR TITLE
Fixes issue-1371: Make the picture sticky in About page

### LINKED ISSUE #1371: Make the picture sticky in About page
I think we should the pictures sticky in the About page. I will try to prompt it.

When we scroll down, half of the page will be blank now. I looks weird.

### PR DESCRIPTION
**Pull request by lixiaoqity**
## Issue This PR Addresses
Fix #1371 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

In About page, when we scroll down, half of the page will be blank now. It looks weird. So, I want to make the pictures sticky.

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by PedroFonsecaDEV — CHANGES_REQUESTED** [review]
Hi @lixiaoqity,
Our About page needs some improvements for sure. 
Be aware of what is going on [1232](https://github.com/Seneca-CDOT/telescope/issues/1232) and [1335](https://github.com/Seneca-CDOT/telescope/issues/1355)
I think it's a better idea to contact @Wei-J-Huang and see how you can manage your ideas together. 

Besides that, the content is running underneath the navbar:

**Review by humphd — CHANGES_REQUESTED** [review]
This isn't working the way I'd expect, since it gets covered by the navbar as I scroll:

sticky-image

**Comment by lixiaoqity** [comment]
I have made the pictures a little below the navbar so that it can not be covered.

File: src/frontend/src/templates/template.js [hunk 1]
```diff
@@ -45,6 +45,10 @@ const useStyles = makeStyles((theme) => ({
     maxWidth: '785px',
     color: theme.palette.text.default,
   },
+  sticky: {
+    position: 'sticky',
+    top: '80px',
+  },
   carousel: {
```
Comment (PedroFonsecaDEV) [inline src/frontend/src/templates/template.js:52]: 1)Find a good position for the carousel, vertically centred. Then after this, we don't want this carousel moving.

2)Let's scroll only the text part.

**Review by PedroFonsecaDEV — CHANGES_REQUESTED** [review]

File: src/frontend/src/templates/template.js (hunk 1 above)
Comment (lixiaoqity) [reply src/frontend/src/templates/template.js:48]: Now the position is moved to the center and fixed. Only text can be scrolled.

File: src/frontend/src/templates/template.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ const tutorialSteps = [
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    flexGrow: 1,
+    // flexGrow: 1,
```
Comment (PedroFonsecaDEV) [inline src/frontend/src/templates/template.js:40]: Please remove this comment. Then I think we are good to go.

**Review by PedroFonsecaDEV — CHANGES_REQUESTED** [review]
We need just to remove one comment line that you left. And I think we will be good to go.

File: src/frontend/src/templates/template.js (hunk 2 above)
Comment (lixiaoqity) [reply src/frontend/src/templates/template.js]: Sorry, forget to remove it. I have fixed it. Thank you.

**Review by PedroFonsecaDEV — APPROVED** [review]

**Review by humphd — CHANGES_REQUESTED** [review]
At smaller window heights, this overflows into the footer.

Screen Shot 2020-11-18 at 8 43 57 PM

**Comment by PedroFonsecaDEV** [comment]
@humphd sorry, I forgot to test all the responsive sizes.

**Review by PedroFonsecaDEV — CHANGES_REQUESTED** [review]
@lixiaoqity 

You need to make the page responsive. 
Screen Shot 2020-11-18 at 8 55 04 PM
Screen Shot 2020-11-18 at 8 55 33 PM
Screen Shot 2020-11-18 at 8 56 01 PM



In case you need the documentation: [material-ui-doc](https://material-ui.com/customization/breakpoints/)

**Comment by lixiaoqity** [comment]
I put the pictures a little close to the above. If it is set on the vertical centre, it will cover the footer.

**Comment by PedroFonsecaDEV** [comment]
We are still having problems.

**Comment by lixiaoqity** [comment]
The two columns look uncomfortable when the screen size is small. Now I think it is better.

**Comment by lixiaoqity** [comment]
I pulled a new request at #1486. Please check it. Thanks.

**Comment by PedroFonsecaDEV** [comment]
image

**Comment by lixiaoqity** [comment]
@PedroFonsecaDEV 
I test the website at the Samsung Galaxy S7 AND iPhone 7 cell phone. It looks well and is different with your screen. My test website domain is "https://telescope-gu1hxh24k.vercel.app/about" which is the latest Vercel check and is different with your test. Can you use it to test?

**Comment by PedroFonsecaDEV** [comment]
@lixiaoqity Sorry, I must have clicked on a different build. 

Please rebase (there is a conflict) so we can check the build after a rebase.
Thanks.

**Comment by lixiaoqity** [comment]
This pull request should be discarded. I have created a new pull request #1486 which is rebased. Thanks.

**Pull request closed without merging**